cypress/e2e/pages/index.spec.ts

```diff
@@ -190,19 +196,23 @@ context("/ on iphone-x", () => {
         .should("be.visible")
         .should("have.text", "Latest Blocks");
 
-      cy.findAllByTestId("BlockCardDetails").within(() => {
-        cy.findByTestId("BlockCardDetails.height").should("be.visible");
-        cy.findByTestId("BlockCardDetails.age").should("be.visible");
-        cy.findByTestId("BlockCardDetails.MintedByLabel")
-          .should("be.visible")
-          .should("have.text", "Minted by");
-        cy.findByTestId("BlockCardDetails.MintedByValue").should("be.visible");
-        cy.findByTestId("BlockCardDetails.TransactionsLabel")
-          .should("be.visible")
-          .should("have.text", "Transactions");
-        cy.findByTestId("BlockCardDetails.TransactionsValue")
-          .should("be.visible")
-          .contains(/\d+/);
+      cy.findAllByTestId("BlockCardDetails").each(($el) => {
+        cy.wrap($el).within(() => {
+          cy.findByTestId("BlockCardDetails.height").should("be.visible");
+          cy.findByTestId("BlockCardDetails.age").should("be.visible");
+          cy.findByTestId("BlockCardDetails.MintedByLabel")
+            .should("be.visible")
+            .should("have.text", "Minted by");
+          cy.findByTestId("BlockCardDetails.MintedByValue").should(
+            "be.visible"
```

Comment:
Can we make this into a single line like this `.should("be.visible");` to maintain its readability